src/main/java/com/entropy/entropay/auth/config/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,27 @@
+package com.entropy.entropay.auth.config;
+
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.Serializable;
+
+/**
```

Comment:
@reconeazaretzky do we have an Entropy CR to add to all classes?

Reply:
@umarcostuc nope. Should we add one? That doesn't conflicts with the license?

Reply:
it would be nice to have one (we should be proud of this state of the art code? lol), it shouldn't conflict with the license but we'd need someone from legal to write it to be sure that it's expending to all public ones we're using.